core/src/generated/mod.rs

```diff
@@ -0,0 +1,4 @@
+// Automatically generated mod.rs
```

Comment:
Note that we don't use `mod.rs` files anywhere else within this repository. Is this necessary to function or would a `generated.rs` file one level above work as well?

Reply:
```
❯ pb-rs --help
pb-rs 0.10.0
Johann Tuffe <tafia973@gmail.com>
A converter from proto files into quick-protobuf compatible Rust module

USAGE:
    pb-rs [FLAGS] [OPTIONS] [INPUT]...

FLAGS:
        --add-deprecated-fields    Add deprecated fields and mark them as #[deprecated]
    -e, --error-cycle              Error out if recursive messages do not have optional fields
    -D, --dont_use_cow             Don't use Cow for String and Byte types
        --gen-info                 Generate MessageInfo implementations
        --hashrown                 Use hashrown for HashMap implementation
        --nostd                    Generate no_std compliant code
    -H, --no-headers               Do not add module comments and module attributes in generated file
    -n, --no-output                Show enums and messages in this .proto file, including those imported. No code
                                   generated
        --owned                    Generate Owned structs when the proto stuct has a lifetime
    -s, --single-mod               Omit generation of modules for each package when there is only one package
    -h, --help                     Prints help information
    -V, --version                  Prints version information

OPTIONS:
    -R, --custom_repr <CUSTOM_REPR>                      The value to use for the optional #[repr(...)] for every struct
    -C, --custom_struct_derive <CUSTOM_STRUCT_DERIVE>
            The comma separated values to add to #[derive(...)] for every struct

    -I, --include <INCLUDE_PATH>                         Path to search for imported protobufs
    -o, --output <OUTPUT>
            Generated file name, defaults to INPUT with 'rs' extension, cannot be used with --output_directory

    -d, --output_directory <OUTPUT_DIR>
            Output directory of generated code, cannot be used with --output


ARGS:
    <INPUT>...    The .proto files used to generate quick-protobuf code
```

There doesn't seem to be an option to not generate a `mod.rs` file.

Reply:
FWIW: I find `mod.rs` more consistent than the libp2p approach, since it keeps all files of a module in the same place.